flow/table.go

```diff
@@ -87,8 +94,9 @@ type Table struct {
 }
 
 // NewTable creates a new flow table
-func NewTable(updateHandler *Handler, expireHandler *Handler, pipeline *EnhancerPipeline) *Table {
+func NewTable(updateHandler *Handler, expireHandler *Handler, pipeline *EnhancerPipeline, nodeTID string, opts TableOpts) *Table {
```

Comment:
OK for options as optional. For nodeTID it is not really an option as it doesnt' really change the way to flow are handled by the flow table